internal/log/log.go

```diff
@@ -1,20 +0,0 @@
-// Copyright Envoy Gateway Authors
-// SPDX-License-Identifier: Apache-2.0
-// The full text of the Apache license is available in the LICENSE file at
-// the root of the repo.
-
-package log
-
-import (
-	"github.com/go-logr/logr"
-	"github.com/go-logr/zapr"
-	"go.uber.org/zap"
-)
-
-func NewLogger() (logr.Logger, error) {
```

Comment:
why delete this ?

Reply:
instead of another `NewLogger` with level that the user setted.